docs/OV_Runtime_UG/multi_device.md

```diff
@@ -1,39 +1,41 @@
+Note that performance increase in this mode comes from utilizing multiple devices at once. This means that you need to provide the devices with enough inference requests to keep them busy or you will not benefit much from using MULTI.
 
-The setup of Multi-Device can be described in three major steps:
 
-1. Prepare configure for each device. 
-2. Compile the model on the Multi-Device plugin created on top of a (prioritized) list of the configured devices with the configure prepared in step one.
-3. As with any other CompiledModel call (resulting from `compile_model`), you create as many requests as needed to saturate the devices.
+## Using the Multi-Device Mode 
 
-These steps are covered below in detail.
+Following the OpenVINO™ naming convention, the Multi-Device mode is assigned the label of “MULTI.” The only configuration option available for it is a prioritized list of devices to use:
 
-### Defining and Configuring the Multi-Device Plugin
+@sphinxdirective
 
-Following the OpenVINO™ convention of labeling devices, the Multi-Device plugin uses the name "MULTI". The only configuration option for the Multi-Device plugin is a prioritized list of devices to use:
++---------------------------+---------------------------------+-----------------------------------------------------------+
+| Property                  | Property values                 | Description                                               |
++===========================+=================================+===========================================================+
+| <device candidate list>   | | AUTO: <device names>          || Specifies the devices available for selection.           |
+|                           | | comma-separated, no spaces    || The device sequence will be taken as priority            |
++---------------------------+---------------------------------+| from high to low.                                        |
+| ov::device::priorities    | | device names                  || Priorities can be set directly as a string.              |
+|                           | | comma-separated, no spaces    |                                                           |
++---------------------------+---------------------------------+-----------------------------------------------------------+
 
-| Parameter name | Parameter values | Default | Description |
-| -------------- | ---------------- | --- | --- |
-| ov::device::priorities | comma-separated device names with no spaces | N/A | Prioritized list of devices |
+@endsphinxdirective
 
-You can set the priorities directly as a string.
+The device candidate list allows users to customize the priority and limit the choice of devices available to MULTI. If not specified, the plugin assumes all the devices present in the system can be used. Importantly, the list may also specify the number of requests for MULTI to keep for each device, as described below.
```

Comment:
" If not specified, the plugin assumes all the devices present in the system can be used"- this is not true, unlike the AUTO, the MULTI does require the explicit list of the devices

Reply:
pls see above, the list is not "candidates".

Reply:
Oh, ok, that is an important difference. Repairing it now.